Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

Enforces usage of /var instead of /usr for data.
Enforces lock file into /var/run.

diff --git a/docs/conf/extra/httpd-dav.conf.in b/docs/conf/extra/httpd-dav.conf.in
index f1d35e0..9961784 100644
--- a/docs/conf/extra/httpd-dav.conf.in
+++ b/docs/conf/extra/httpd-dav.conf.in
@@ -12,18 +12,18 @@
 # on the directory where the DavLockDB is placed and on any directory where
 # "Dav On" is specified.
 
-DavLockDB "@@ServerRoot@@/var/DavLock"
+DavLockDB "@@RuntimeDir@@/DavLock"
 
-Alias /uploads "@@ServerRoot@@/uploads"
+Alias /uploads "@@DataDir@@/uploads"
 
-<Directory "@@ServerRoot@@/uploads">
+<Directory "@@DataDir@@/uploads">
     Dav On
 
     AuthType Digest
     AuthName DAV-upload
     # You can use the htdigest program to create the password database:
-    #   htdigest -c "@@ServerRoot@@/user.passwd" DAV-upload admin
-    AuthUserFile "@@ServerRoot@@/user.passwd"
+    #   htdigest -c "@@SysconfDir@@/user.passwd" DAV-upload admin
+    AuthUserFile "@@SysconfDir@@/user.passwd"
     AuthDigestProvider file
 
     # Allow universal read-access, but writes are restricted
